backend/lib/search.py

```diff
@@ -112,8 +114,11 @@ def process(self):
 				# file exists somewhere, so we create it as an empty file
 				with open(query_parameters.get("copy_to"), "w") as empty_file:
 					empty_file.write("")
-
-		self.dataset.finish(num_rows=num_items)
+		if self.flawless:
+			self.dataset.finish(num_rows=num_items)
+		else:
+			self.dataset.update_status("Some items did not map correctly. Data is maintained, but will not be available in 4CAT; check logs for details", is_final=True)
```

Comment:
I'd make this a bit more specific and unambiguous, e.g. "[number] items could not be parsed. These remain in the [extension] file, but will not be available to 4CAT processors or in the CSV download. Check the log file for details"

Reply:
Hmmm. The specific items are available in the dataset log, but I can look at counting them. Perhaps flawless could be an int and 0 actually represents "flawless".